src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -1,13 +1,15 @@
-<div id="sfwdt{{ token }}" class="sf-toolbar" style="display: none"></div>
```

Comment:
As you want to have it merged as a bug fix in 2.7, it should avoid doing any unnecessary changes (which have a risk of having a negative impact while providing no value)